bin/validate_ami_bags.py

```diff
@@ -17,112 +14,143 @@ def _configure_logging(args):
     else:
         logging.basicConfig(level=logging.INFO, format=log_format)
 
-
 def _make_parser():
-    parser = argparse.ArgumentParser()
-    parser.description = "check if an AMI bag meets specifications"
-    parser.add_argument("-d", "--directory",
-                        nargs='+',
-                        help = "Path to a directory full of bags")
-    parser.add_argument("-b", "--bagpath",
-                        nargs='+',
-                        default = None,
-                        help = "Path to the base directory of the bag")
-    parser.add_argument("--slow", action='store_false',
-                        help = "Recalculate hashes (very slow)")
-    parser.add_argument("--metadata", action='store_true',
-                        help = "Validate Excel metadata files")
+    parser = argparse.ArgumentParser(description="Check if an AMI bag meets specifications")
+    parser.add_argument("-d", "--directory", nargs='+', help="Path to a directory full of bags")
+    parser.add_argument("-b", "--bagpath", nargs='+', default=None, help="Path to the base directory of the bag")
+    parser.add_argument("--slow", action='store_false', help="Recalculate hashes (very slow)")
+    parser.add_argument("--metadata", action='store_true', help="Validate Excel metadata files")
     parser.add_argument('--log', help='The name of the log file')
     parser.add_argument('-q', '--quiet', action='store_true')
     return parser
 
-
-
-def main():
-    parser = _make_parser()
-    args = parser.parse_args()
-
-    bags = []
-
-    _configure_logging(args)
-
+def log_checks(args):
     checks = """Performing the following validations:
-    Checking 0xums,
+    Checking Oxums,
```

Comment:
Changed this back to "Oxums" (that's the default, checksums only checked when --slow )